### DISCUSSION TITLE
Transmission for private file distribution between servers

### DISCUSSION THREAD

**Discussion by FlaviuRadulescu** [transmission/transmission#6002] [General]
Please excuse me if this question is stupid. I am new to transmission and haven't found anything relevant in the documentation related to my question.

I am wondering if I can use transmission on 5 servers to sync files between these. I don't need a user interface, so would be great to be able to configure it one time in a config file. I just want to configure it in such a way that when I put a file into a folder will automatically distribute to all the other servers using the BitTorrent protocol. The files never changed, they are added or deleted, and the file sizes are from 100 MB to 10 GB

Could I do this with Transmission? Any documentation or help on how to do this would be greatly appreciated

**Comment by NiklasBr** [comment]
You should take a look at https://github.com/Jacalz/rymdport
